dynamic-layers/toradex-ti-layer/recipes-kernel/linux/linux-toradex-ti/verdin-am62-hmm/0001-add-hmm-device-tree.patch

```diff
@@ -664,6 +680,13 @@ index 000000000000..b8cd6c7666a7
 +			AM62X_IOPAD(0x009c, PIN_INPUT, 7) /* (V25) GPMC0_WAIT1.GPIO0_38 */ /* GPIO_SODIMM_189 */
 +		>;
 +	};
++	/* Accelerometer IRQ line */
++	pinctrl_acc: acc_grp {
++		pinctrl-single,pins = <
++			AM62X_MCU_IOPAD(0x000c, PIN_INPUT_PULLUP, 7) /* (D9) MCU_SPI0_D0.MCU_GPIO0_3 */ /* GPIO_SODIMM 210 */
```

Comment:
Have you checked the data scheet of the accl how the INT1 output is constructed?